docs/dev-guide.md

```diff
@@ -36,45 +43,47 @@ This topic describes how to contribute to the Tableau Server Client (Python) pro
    git checkout -b 13-feature-new-stuff
    ```
 
-1. Code and commit!
+### Code and commit
 
-   Here's a quick checklist for ensuring a good pull request:
+   Here's a quick checklist to follow when coding to ensure a good pull request (PR):
 
-   - Only touch the minimal amount of files possible while still accomplishing the goal.
+   - Only touch the fewest number of files possible while still accomplishing the goal.
    - Ensure all indentation is done as 4-spaces and your editor is set to unix line endings.
-   - The code matches PEP8 style guides. If you cloned the repo you can run `pycodestyle .`
+   - The code matches PEP8 style guides. If you cloned the repo you can run `pycodestyle server-client-python test samples`
```

Comment:
this should be `pycodestyle tableauserverclient test samples`

Reply:
Updated the `pycodestyle` command as suggested.